### ISSUE TITLE
Unable to Compute Indices for Large Region (~9 Lakh Acres) Using Farmvibes and Sentinel-2 Data

### ISSUE THREAD

**Issue by click2cloud-sagarB** [microsoft/farmvibes-ai#117]
Dear Farmvibes Team,

I looked at the sentinel2 official paperwork, and it states that each tile has a surface area of 110x110km2 i.e 2 989 975.12 acres (~29 lakh acres). But I received Runtime error message "No product group can cover input geometry" when I attempted to download tiles for even a region of 9 lakh acres. But in sentinel2 official paperwork it is mentioned user product  covers 110x110km² i.e 2 989 975.12 acres (29 lakh acres).
But using farmvibes we are not able to compute indices for even 9 lakh acres region.
For this workflow, I'm utilising the entire year 2022 as the timerange.
Link for official paperwork I used https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-2/data-products#:~:text=However%2C%20each%20tile%20has%20a,Earth%27s%20surface%20into%2060%20zones

Attached to this issue, you will find the following files for your reference:
1) Log_file for failure
2) .wkt file of the farm for which I got the error

[files.zip](https://github.com/microsoft/farmvibes-ai/files/12240006/files.zip)

**Comment by rafaspadilha** [comment]
Hi, @click2cloud-sagarB. Thank you for raising this issue.

The error you are seeing happens when we list and select the available Sentinel-2 products to find those that can cover the desired input geometry. Even though the tile is larger than your input geometry, we do not have any guarantees that the geometry will be entirely covered by one (or more) products.

I've plotted a few products that intersect with the shared geometry in that time range. It seems that our geometry (in orange) is never fully covered by available products' geometries (in blue). I've found two swap patterns for different dates:

![b2e04b4f-4062-4e6f-981f-09c814916200](https://github.com/microsoft/farmvibes-ai/assets/6332958/87f514e8-4a4d-4fa2-8e5a-5c788f9207be)

and

![450e2e82-4468-4271-8d73-2785282c3864](https://github.com/microsoft/farmvibes-ai/assets/6332958/e2f64924-8344-4fd0-88b1-0a9d357deba5)


Because of this, FarmVibes cannot provide a raster and the error is raised. A possible workaround to this would be running the workflows with a smaller geometry.

Hope this clarifies the issue.
